plugins/vendor/zoom/lib/api/webPage/KalturaZoomRegistrationPage.html

```diff
@@ -1,136 +1,49 @@
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
     <script>
         var ks = '@ks@';
-        var url = '@BaseServiceUrl@/api_v3/service/vendor_zoomvendor/action/submitRegistration';
-        var accountId = '@accountId@';
+        let baseServiceUrl = '@BaseServiceUrl@';
+        var accountId = '@accoutId@';
+        var partnerId= '@partnerId@';
+        var submitUrl = baseServiceUrl+'/api_v3/service/vendor_zoomvendor/action/submitRegistration';
+        var getUrl = baseServiceUrl + '/api_v3/service/vendor_zoomvendor/action/get?format=1';
+
     </script>
     <script>
-        function trySubmit(event) {
+        function trySubmit() {
             $("body").css("cursor", "progress");
-            event.preventDefault();
-            var $defaultUserId = $('#defaultUserId');
-            var $zoomCategory  = $('#zoomCategory');
-            var $zoomWebinarCategory  = $('#zoomWebinarCategory');
-            var defaultUserId = $defaultUserId.val().trim();
-            var zoomCategory = $zoomCategory.val().trim();
-            var zoomWebinarCategory = $zoomWebinarCategory.val().trim();
-            var enableRecordingUploadObj = $(".enableRecordingUpload");
-            var enableWebinarUploadObj = $(".enableWebinarUpload");
-            var createUserIfNotExistObj = $(".createUserIfNotExist");
-            var enableRecordingUpload = enableRecordingUploadObj.is(":checked");
-            var enableWebinarUpload = enableWebinarUploadObj.is(":checked");
-            var createUserIfNotExist = createUserIfNotExistObj.is(":checked");
-            var handleParticipantsMode = $('[name=participantsHandler]:checked').val();
-            var zoomUserMatchingMode = $('[name=userMatchingHandler]:checked').val();
-            var zoomUserPostfix = $('#zoomUserPostfix').val().trim();
+            let defaultUserId = $('#defaultUserId').val().trim();
+            let zoomCategory  = $('#zoomCategory').val().trim();
+            let zoomWebinarCategory  = $('#zoomWebinarCategory').val().trim();
+            let enableRecordingUpload = $("#enableRecordingUpload").is(":checked");
+            let enableWebinarUpload = $("#enableWebinarUploads").is(":checked");
+            let handleParticipantsMode = $('[name=handleParticipantsMode]:checked').val();
+            let zoomUserMatchingMode;
+            if($('[name=recordingOwner]:checked').val()==1)
+            {
+                zoomUserMatchingMode = 3;
+            }
+            else
+            {
+                zoomUserMatchingMode = $('[name=zoomUserMatchingMode]:checked').val();
```

Comment:
why not initialize zoomUserMatchingMode with $('[name=zoomUserMatchingMode]:checked').val(); and override only if the 'if' condition satisfied?